src/utils/generator.ts

```diff
@@ -546,7 +546,7 @@ export function adjustBarrelIndex(
         }
       }
 
-      if (type === "component" || type === "service") {
+      if (type === "component" || type === "service" || type === "directive" || type === 'pipe') {
```

Comment:
@m-abs thank you for this PR and good catch. This is the only change I’m not so sure about. There’s really never a case I’ve run into where I need the directive or pipe symbol exported from the barrel which is why this was just component and service since those are often used for various reasons. Curious what your reasons are for this change?

Reply:
Mostly it was to be consistent with components, like here: https://github.com/nstudio/xplat/blob/master/src/feature/_nativescript_component_files/components/index.ts#L7

We sometimes use pipes in code, so it makes sense to have those exposed.

Reply:
I removed it from directives but kept it for pipes.

Reply:
@m-abs perfect thanks! That makes sense for pipes yeah since can be used as provider where needed.